crates/hotshot/src/traits/networking/web_server_network.rs

```diff
@@ -231,6 +233,182 @@ struct Inner<TYPES: NodeType> {
 
 impl<TYPES: NodeType> Inner<TYPES> {
     #![allow(clippy::too_many_lines)]
+
+    /// Handle version 0.1 transactions
+    async fn handle_tx_0_1(&self, tx: Vec<u8>, index: u64, tx_index: &mut u64) {
```

Comment:
Could you add a comment for the `index` and `tx_index` arguments? Not so easy to guess what the difference is. 🤔

Reply:
Added!

I think `index` is supposed to be the first transaction returned by the server, and `tx_index` is the last transaction read.

(But let me know if this seems wrong! I'm mostly guessing from the context where they were used.)